src/main/java/no/fdk/listener/RabbitMQListener.java

```diff
@@ -26,8 +26,9 @@ private void receiveMessage(Message message) {
             .flatMapMany(graph -> assessmentService.assess(graph, EntityType.DATASET))
             .doOnNext(assessmentService::upsertAssessment)
             .count()
+            .onErrorReturn(0L)
```

Comment:
Den vil nå acke meldingen i køen til rabbit?

Reply:
Ikke denne linjen direkte. Denne er kun en fallback for count funksjonen. Dersom fallback er ikke oppgitt, vil den bruke egen fallback, som printer ting til loggen - ikke pent.